sdk/communication/Azure.Communication.Chat/src/Models/ChatModelFactory.cs

```diff
@@ -47,13 +47,5 @@ public static ChatThreadInfo ChatThreadInfo(string id, string topic, DateTimeOff
         /// <returns>A new <see cref="ChatMessageReadReceipt"/> instance for mocking.</returns>
         public static ChatMessageReadReceipt ChatMessageReadReceipt(string senderId, string chatMessageId, DateTimeOffset? readOn)
```

Comment:
These should be hidden from Intellisense using EditorBrowsable attribute.
E.g. - https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/storage/Azure.Storage.Blobs/src/Models/BlobsModelFactory.cs#L20

Reply:
Why?